### DISCUSSION TITLE
Jira에 새 스프린트가 추가된 것을 보았습니다. 그 스프린트를 시작시킬 순 없던가요?

### DISCUSSION THREAD

**Discussion by Woon-2** [MyeongWoonJang/cpp-study#10] [Q&A] [answered]
![백로그](https://user-images.githubusercontent.com/73771162/191648293-2260e301-b6b1-4558-b722-0abfe5b20ac5.PNG)

Jira에 새 스프린트가 추가된 것을 보았습니다. 그 스프린트를 시작시킬 순 없던가요?

원래 스프린트는 하나씩 진행하는 게 맞긴 한데, 만들어진 걸 보니 동시 진행도 가능한가 호기심이 생겨서요.

저는 스프린트 시작 버튼이 비활성화되어있는데, 제가 만든 스프린트가 아니라서 그런지, 이미 진행되고 있는 스프린트가 있어서 그런지 모르겠습니다.

___

그리고 Issue에 우선 순위 항목이 추가되었더군요.
매우 굿입니다.
저는 모르던 기능이었어요.

**Comment by Lcat12345** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/62102830/191758645-634db01e-07c5-47e1-9b52-d8fde5e14d6f.png)
저 역시 비활성화 되어있습니다.

**Reply by Woon-2** [reply]
단일 스프린트만 진행 가능한 것 같습니다. 감사합니다.
